.github/CODEOWNERS

```diff
@@ -1,3 +1,3 @@
 # https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners
-* @openfga/backend
+* @openfga/backend @openfga/dx
```

Comment:
I'll revert this prior to merge so that I can still easily get any changes into the `feat/stronger-consistency-options` branch if needed